### ISSUE TITLE
Conflict when using 'esm' module

### ISSUE THREAD

**Issue by tracker1** [gulpjs/findup-sync#45]
trying to use this in conjunction with the 'esm' module..

```
node -r esm

> var fu = require('findup-sync')
```

drops out of node at time of require.

**Comment by phated** [comment]
Hey @tracker1, been awhile.  Do you have any more debug info?

**Comment by tracker1** [comment]
@phated not really, took me a while just to narrow it down to the issue I was having...  I was only looking for package.json, and for now, just switched to use path.join(process.cwd, 'package.json') which works for my needs at the moment.  It's really weird and that the whole world just seems to stop, and poof, no longer inside node.

@phated Need to get to a nodeaz meetup soon, been too long.

**Comment by OmgImAlexis** [comment]
I think I've found the issue.

Seems `esm` isn't playing nice with [`https://github.com/jonschlinkert/resolve-dir`](https://github.com/jonschlinkert/resolve-dir).
